### DISCUSSION TITLE
Easy way to host exported glb scenes in our hubs cloud instances?

### DISCUSSION THREAD

**Discussion by matthewbcool** [mozilla/hubs#2758] [General Questions & Discussion] [answered]
From spoke, the easiest way I've found to export into my scenes is by publishing to hubs and then using the link. Are there any other good routes if we do not wish to publish to hubs?

**Comment by matthewbcool** [comment]
Also curious about best practices for versioning. For example: I make improvements to scene and now do I just republish to hubs? Am I overwriting the original? Or how can I use the current system to just point to the new scene. Would love to hear how others are doing it.

**Reply by emclaren** [reply]
Hi @matthewbcool - for versioning, there is a Save As button inside of Spoke, I find it helpful if I'm making a big change to a scene to save a new copy of it. When you re-publish a scene you are overwriting the scene. Any rooms that already have that scene loaded will not be automatically updated (they will still look like the old scene), but if you go to "choose a scene", and pick the scene from the updated list, it will overwrite with the new scene content.

**Reply by matthewbcool** [reply]
super helpful responses! Thank you for the clarification.

**Comment by emclaren** [comment, marked as answer]
If you don't want to publish to Hubs, there should be an option to export your scene as a .glb file or a .spoke file. this would allow you to save a version of the scene without publishing to Hubs. You won't be able to open them as a Hubs room without publishing them somehow though. 

You should have a version of Spoke on your own Hubs Cloud instance if you don't want to publish on hubs.mozilla.com. To access this go to [your-hubs-cloud-url]/spoke

Just so you know, if you do publish your scene on hubs.mozilla.com, the scene is private to you unless you check off the boxes that say that the scene is remixable and promotable when you publish it. It will only be seen by people with whom you share the link to the scene.

**Comment by vernsworld** [comment]
Not sure where to find this info and would greatly appreciate a push in the right direction, but where can I find info about exporting a spoke scene and how to host it on my own server?

**Reply by matthewbcool** [reply]
Hi @vernsworld!  If you are hosting your own hubs cloud, you can import your spoke scene by using the scene URL in the admin panel. The scene link is the link that looks like `https://hubs.mozilla.com/scenes/hAEAos4`. There are a couple of other export options in the file menu of spoke as well. Hope that helps!


<img width="1540" alt="importadmin" src="https://user-images.githubusercontent.com/4493657/113005114-c293d900-91a6-11eb-8f54-db21d54f23fd.png">
